beacon-chain/db/kv/lightclient_test.go

```diff
@@ -613,177 +367,107 @@ func TestStore_LightClientUpdate_EmptyDB(t *testing.T) {
-			Version: 1,
-			Data: &ethpbv2.LightClientUpdate{
-				AttestedHeader:          nil,
-				NextSyncCommittee:       nil,
-				NextSyncCommitteeBranch: nil,
-				FinalizedHeader:         nil,
-				FinalityBranch:          nil,
-				SyncAggregate:           nil,
-				SignatureSlot:           primitives.Slot(uint64(i)),
-			},
+
+	for i, update := range updates {
+		if i == 1 || i == 3 {
+			continue
 		}
-		err := db.SaveLightClientUpdate(ctx, uint64(i), update)
+		err := db.SaveLightClientUpdate(ctx, uint64(i+1), update)
 		require.NoError(t, err)
 	}
 
-	// Retrieve the updates from 1 to 5
-	retrievedUpdates, err := db.LightClientUpdates(ctx, 1, 5)
+	// Retrieve the updates
+	retrievedUpdates, err := db.LightClientUpdates(ctx, 0, 7)
 	require.NoError(t, err)
 	require.Equal(t, 3, len(retrievedUpdates))
-	require.Equal(t, primitives.Slot(1), retrievedUpdates[1].Data.SignatureSlot, "retrieved update does not match saved update")
-	require.Equal(t, primitives.Slot(2), retrievedUpdates[2].Data.SignatureSlot, "retrieved update does not match saved update")
-	require.Equal(t, primitives.Slot(3), retrievedUpdates[3].Data.SignatureSlot, "retrieved update does not match saved update")
-
-}
-
-func setupLightClientTestDB(t *testing.T) (*Store, context.Context) {
-	db := setupDB(t)
-	ctx := context.Background()
-
-	for i := 10; i < 101; i++ { // 10 to 100
-		update := &ethpbv2.LightClientUpdateWithVersion{
-			Version: 1,
-			Data: &ethpbv2.LightClientUpdate{
-				AttestedHeader:          nil,
-				NextSyncCommittee:       nil,
-				NextSyncCommitteeBranch: nil,
-				FinalizedHeader:         nil,
-				FinalityBranch:          nil,
-				SyncAggregate:           nil,
-				SignatureSlot:           primitives.Slot(uint64(i)),
-			},
-		}
-		err := db.SaveLightClientUpdate(ctx, uint64(i), update)
-		require.NoError(t, err)
-	}
-
-	for i := 110; i < 201; i++ { // 110 to 200
-		update := &ethpbv2.LightClientUpdateWithVersion{
-			Version: 1,
-			Data: &ethpbv2.LightClientUpdate{
-				AttestedHeader:          nil,
-				NextSyncCommittee:       nil,
-				NextSyncCommitteeBranch: nil,
-				FinalizedHeader:         nil,
-				FinalityBranch:          nil,
-				SyncAggregate:           nil,
-				SignatureSlot:           primitives.Slot(uint64(i)),
-			},
-		}
-		err := db.SaveLightClientUpdate(ctx, uint64(i), update)
-		require.NoError(t, err)
-	}
-
-	return db, ctx
+	require.DeepEqual(t, updates[0], retrievedUpdates[uint64(1)], "retrieved update does not match saved update")
+	require.DeepEqual(t, updates[2], retrievedUpdates[uint64(3)], "retrieved update does not match saved update")
+	require.DeepEqual(t, updates[4], retrievedUpdates[uint64(5)], "retrieved update does not match saved update")
 }
 
-func TestStore_LightClientUpdate_MissingPeriodsInTheMiddleDistributed(t *testing.T) {
-	db, ctx := setupLightClientTestDB(t)
+func createDefaultLightClientUpdate(currentSlot primitives.Slot) (interfaces.LightClientUpdate, error) {
```

Comment:
Weren't you supposed to use the exported `CreateDefaultLightClientUpdate` function instead of creating a copy here?

Reply:
circular import error.